src/bp/ui-studio/src/web/views/Nlu/client.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios'
+import { NLU } from 'botpress/sdk'
+
+export type NluClient = ReturnType<typeof makeNLUClient>
+
+export const makeNLUClient = () => {
+  const client = axios.create({ baseURL: `${window.BOT_API_PATH}/nlu` })
+  return {
+    fetchContexts: (): Promise<string[]> => client.get('/contexts').then(res => res.data),
+    fetchIntentsWithQNAs: (): Promise<NLU.IntentDefinition[]> => client.get('/intents').then(res => res.data),
+    fetchIntents: async (): Promise<NLU.IntentDefinition[]> => {
+      const { data } = await client.get('/intents')
+      return data.filter(x => !x.name.startsWith('__qna__'))
+    },
+    fetchIntent: (intentName: string): Promise<NLU.IntentDefinition> =>
+      client.get(`/intents/${intentName}`).then(res => res.data),
+    createIntent: (intent: Partial<NLU.IntentDefinition>) => client.post('/intents', intent),
+    updateIntent: (targetIntent: string, intent: Partial<NLU.IntentDefinition>): Promise<void> =>
+      client.post(`/intents/${targetIntent}`, intent),
+    deleteIntent: (name: string): Promise<void> => client.post(`/intents/${name}/delete`),
+    syncIntentTopics: (intentNames?: string[]): Promise<void> => client.post('/sync/intents/topics', { intentNames }),
```

Comment:
I don't think this one is used in the frontend (I'm not even sure its used in the backend either)

Reply:
You made me realise I might be breaking ndu stuff, I'll double check